app/views/passengers/index.haml

```diff
@@ -30,9 +30,11 @@
     %thead
       %tr
         %th Name
+        %th Mobility Device
         %th Email
         %th Phone
-        %th.yesno Active
+        - unless @filters.include? 'Active'
```

Comment:
Why is the filter a string?

Reply:
https://github.com/umts/st-pax/blob/3f98d3086a213d1de593a907f785ae44a5dea8a7/app/views/passengers/index.haml#L1-L3

Reply:
I know, but why

Reply:
That *is* why. So that we can join them. I thought about using symbols, but I'd just need to `map(&:to_s)` them first.

See discussion at #34.